certbot/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * Disable old SSL versions and ciphersuites to follow Mozilla recommendations in Apache.
 * Remove ECDHE-RSA-AES128-SHA from NGINX ciphers list now that Windows 2008 R2 and Windows 7 are EOLed
 * Support for Python 3.4 has been removed.
+* Remove SSLCompression off line from all config options
```

Comment:
nit: Can we expand, generalize, and/or combine this changelog entry with the one about SSL versions and ciphersuites?

I think if I was an outsider reading this line, I wouldn't understand the change being made.